src/main/java/org/openrewrite/github/AddCronTrigger.java

```diff
@@ -17,31 +17,68 @@
 
 import lombok.EqualsAndHashCode;
 import lombok.Getter;
+import org.jetbrains.annotations.VisibleForTesting;
 import org.openrewrite.Option;
 import org.openrewrite.Recipe;
 import org.openrewrite.yaml.MergeYaml;
 
 import java.time.Duration;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Random;
+import java.util.concurrent.ThreadLocalRandom;
 
 @EqualsAndHashCode(callSuper = false)
 @Getter
 public class AddCronTrigger extends Recipe {
     @Option(displayName = "Cron expression",
-            description = "Using the [POSIX cron syntax](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/crontab.html#tag_20_25_07).",
-            example = "0 18 * * *")
+            description = "Using the [POSIX cron syntax](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/crontab.html#tag_20_25_07) or the non standard options" +
+                          " @hourly @daily @weekly @weekdays @weekends @monthly @yearly.",
+            example = "@daily")
     private final String cron;
 
-    public AddCronTrigger(String cron) {
-        this.cron = cron;
+    @VisibleForTesting
+    final transient Random random;
+
+    @VisibleForTesting
+    AddCronTrigger(String cron, Random random) {
+        this.random = random;
+        this.cron = parseExpression(cron);
         doNext(new MergeYaml(
                 "$.on",
-                "" +
-                        "schedule:\n" +
-                        "  - cron: \"0 18 * * *\"",
+                String.format("schedule:\n" +
```

Comment:
Wondering if this should be `%n`, but don't know how other recipes handle that.
```suggestion
                String.format("schedule:%n" +
```